src/dotnet/APIView/APIViewWeb/Client/src/review.ts

```diff
@@ -48,14 +48,12 @@ $(() => {
     if (setTo == "OPEN") {
       caretIcon.removeClass("fa-angle-right");
       caretIcon.addClass("fa-angle-down");
-      caretIcon.css("color", "darkorange");
```

Comment:
what is this for?

Reply:
Used to color the expand/collapse icon orange when open. It does not look good in diff scenarios due to different colors (green and red) of the lines